src/CSET/operators/plot.py

```diff
@@ -202,16 +201,23 @@ def _plot_and_save_contour_plot(
         Filename of the plot to write.
     title: str
         Plot title.
-
+    method: "contour" | "pcolormesh"
+        The plotting method to use.
     """
     # Setup plot details, size, resolution, etc.
     fig = plt.figure(figsize=(8, 8), facecolor="w", edgecolor="k")
 
     # Specify the color bar
     cmap, levels, norm = _colorbar_map_levels(cube.name())
 
-    # Filled contour plot of the field.
-    contours = iplt.contourf(cube, cmap=cmap, levels=levels, norm=norm)
```

Comment:
Minor comment, maybe pass method as 'contourf' as filled contours (contourf) are different to non-filled contours (contour). Might make it bit easier down the line if new methods are added to be specific.

Reply:
Fixed.